src/engine/Engine.class.spec.js

```diff
@@ -109,7 +109,7 @@ describe('Engine', () => {
     }]
 
     engine.cache.cacheValues = jest.fn()
-    await engine.addValues('sourceId', sampleValues)
+    await engine.addValues('sourceId', 'sourceName', sampleValues)
```

Comment:
It should be possible to deduct the name from the id? why do we give the 2 parameters?

Reply:
Indeed, we could retrieve it from the "activeApis" or "activeProtocols" lists from the engine